### PR TITLE
Small update to contributing docs

### PR DESCRIPTION
**Pull request by zephraph**
Just a minor note to remind folks to run `shards install` when setting up the project.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -28,6 +28,12 @@ Then, follow [this instruction][2].
 
 [2]: https://crystal-lang.org/docs/installation/on_debian_and_ubuntu.html
 
+### Install the project's crystal dependencies
+
+```
+shards install
```
Comment (Sija) [inline CONTRIBUTING.md:34]:
```suggestion
$ shards install
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -28,6 +28,12 @@ Then, follow [this instruction][2].
 
 [2]: https://crystal-lang.org/docs/installation/on_debian_and_ubuntu.html
 
+### Install the project's crystal dependencies
```
Comment (Sija) [inline CONTRIBUTING.md:31]:
```suggestion
### Installing Mint dependencies
```

**Comment by zephraph** [comment]
Thanks! Updated.

**Comment by s0kil** [comment]
@zephraph Thank You 🥬

**Pull request merged**